Description: Use "python -m covergate" instead of just "coverage"
 In Debian, there's no "coverage" command, so we need to apply this.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-03-14

--- horizon-2014.1~b3.orig/run_tests.sh
+++ horizon-2014.1~b3/run_tests.sh
@@ -315,8 +315,8 @@ function run_tests_all {
     export NOSE_HTML_OUT_FILE='horizon_nose_results.html'
   fi
   if [ $with_coverage -eq 1 ]; then
-    ${command_wrapper} coverage erase
-    coverage_run="coverage run -p"
+    ${command_wrapper} python -m coverage erase
+    coverage_run="python -m coverage run -p"
   fi
   ${command_wrapper} ${coverage_run} $root/manage.py test horizon --settings=horizon.test.settings $testopts
   # get results of the Horizon tests
@@ -333,9 +333,9 @@ function run_tests_all {
 
   if [ $with_coverage -eq 1 ]; then
     echo "Generating coverage reports"
-    ${command_wrapper} coverage combine
-    ${command_wrapper} coverage xml -i --include="horizon/*,openstack_dashboard/*" --omit='/usr*,setup.py,*egg*,.venv/*'
-    ${command_wrapper} coverage html -i --include="horizon/*,openstack_dashboard/*" --omit='/usr*,setup.py,*egg*,.venv/*' -d reports
+    ${command_wrapper} python -m coverage combine
+    ${command_wrapper} python -m coverage xml -i --include="horizon/*,openstack_dashboard/*" --omit='/usr*,setup.py,*egg*,.venv/*'
+    ${command_wrapper} python -m coverage html -i --include="horizon/*,openstack_dashboard/*" --omit='/usr*,setup.py,*egg*,.venv/*' -d reports
   fi
   # Remove the leftover coverage files from the -p flag earlier.
   rm -f .coverage.*
